dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeSdk.kt

```diff
@@ -8,40 +8,67 @@ package com.datadog.android.bridge.internal
 
 import android.content.Context
 import android.util.Log
-import com.datadog.android.Datadog as DatadogSDK
 import com.datadog.android.bridge.DdSdk
 import com.datadog.android.bridge.DdSdkConfiguration
 import com.datadog.android.core.configuration.Configuration
 import com.datadog.android.core.configuration.Credentials
 import com.datadog.android.privacy.TrackingConsent
-import com.datadog.android.rum.GlobalRum
 import com.datadog.android.rum.RumMonitor
 
-internal class BridgeSdk(context: Context) : DdSdk {
+internal class BridgeSdk(
+    context: Context,
+    private val datadog: DatadogWrapper
+) : DdSdk {
 
     internal val appContext: Context = context.applicationContext
 
+    // region DdSdk
+
     override fun initialize(configuration: DdSdkConfiguration) {
-        val credentials = Credentials(
-            clientToken = configuration.clientToken,
-            envName = configuration.env,
-            rumApplicationId = configuration.applicationId,
-            variant = ""
-        )
-        val configBuilder = Configuration.Builder(
+        val credentials = buildCredentials(configuration)
+        val nativeConfiguration = buildConfiguration(configuration)
+
+        datadog.setVerbosity(Log.VERBOSE)
+        datadog.initialize(appContext, credentials, nativeConfiguration, TrackingConsent.GRANTED)
```

Comment:
I guess we will have a task to set also the `trackingConsent`

Reply:
Yes this is scheduled ;)